rules/REQUEST-921-PROTOCOL-ATTACK.conf

```diff
@@ -84,7 +84,7 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
     setvar:'tx.anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
 
 
-SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/* "@rx (?:\bhttp\/(?:0\.9|1\.[01])|<(?:html|meta)\b)" \
+SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/* "@rx (?:\bhttp\/\d|<(?:html|meta)\b)" \
```

Comment:
I think there is an extra `\` character here before the `/\d`. In 3.3 the operator's argument is [this](https://github.com/coreruleset/coreruleset/blob/v3.3/master/rules/REQUEST-921-PROTOCOL-ATTACK.conf#L89):

```
(?:\bhttp/\d|<(?:html|meta)\b)
```
(and not `(?:\bhttp\/\d|<(?:html|meta)\b)`)